app/src/main/java/de/westnordost/streetcomplete/about/CreditsFragment.kt

```diff
@@ -71,8 +72,12 @@ class CreditsFragment : Fragment(R.layout.fragment_credits) {
     }
 
     private suspend fun readCodeContributors() = withContext(Dispatchers.IO) {
-        resources.getYamlObject<List<String>>(R.raw.credits_code).map(::withLinkToGithubAccount) +
-            getString(R.string.credits_and_more)
+        // because they are mentioned as "main contributors" already
+        val skipUsers = setOf("westnordost", "FloEdelmann", "matkoniecz", "ENT8R")
```

Comment:
@westnordost I took the liberty to implement and commit this right now, since I have also asked myself the same thing during my review. See [`242eb99`](https://github.com/streetcomplete/StreetComplete/pull/3850/commits/242eb99fed27218900bae1ca801f0db8902dcb5d).